assets/memberships-gate/editor.js

```diff
@@ -2,22 +2,36 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __ } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { compose } from '@wordpress/compose';
 import { Fragment, useEffect } from '@wordpress/element';
-import { Button, TextControl, CheckboxControl } from '@wordpress/components';
+import { Button, TextControl, CheckboxControl, SelectControl } from '@wordpress/components';
 import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
 import { registerPlugin } from '@wordpress/plugins';
 
 /**
  * Internal dependencies
  */
+import PositionControl from '../components/src/position-control';
 import './editor.scss';
 
 const styles = [
-	{ name: 'inline', label: __( 'Inline', 'newspack' ) },
-	{ name: 'overlay', label: __( 'Overlay (soon)', 'newspack' ) },
+	{ value: 'inline', label: __( 'Inline', 'newspack' ) },
+	{ value: 'overlay', label: __( 'Overlay', 'newspack' ) },
+];
+
+const overlayPositionsLabels = {
+	center: __( 'center', 'newspack' ),
+	bottom: __( 'bottom', 'newspack' ),
+};
+
+const overlaySizes = [
+	{ value: 'x-small', label: __( 'Extra Small', 'newspack' ) },
+	{ value: 'small', label: __( 'Small', 'newspack' ) },
+	{ value: 'medium', label: __( 'Medium', 'newspack' ) },
+	{ value: 'large', label: __( 'Large', 'newspack' ) },
+	{ value: 'full', label: __( 'Full Width', 'newspack' ) },
```

Comment:
```suggestion
	{ value: 'full-width', label: __( 'Full Width', 'newspack' ) },
```

If you set this value to `full-width`, you get this nice change in the style of the `PositionControl`:

<img width="270" alt="Screen Shot 2023-03-31 at 2 27 56 PM" src="https://user-images.githubusercontent.com/2230142/229223453-980fbb2e-4c1d-471e-9610-7b79f8003985.png">